spec/core.md

```diff
@@ -18,187 +18,253 @@ DID, after the prefix, is the case-sensitive [[ref: method-specific identifier]]
 
 As with `did:web`, this method reads data from whatever web server is referenced
 when the (host+)domain portion of one of its DIDs is resolved through the Domain
-Name System (DNS).
+Name System (DNS). In fact, a `did:webs` can be resolved to a [[ref: DIDDoc]]
+using a simple text transformation to an HTTPS URL in exactly the same way as a
+`did:web`. Thus, a `did:web` and `did:webs` with the same [[ref: method-specific
```

Comment:
I think our resolution rules are actually a superset of did:web's rules, so I'd replace "exactly the same way" with just "the same way" and I'd change "Thus," to "By design," because I don't think we've said enough for the reader to see that this follows logically.

Reply:
Fixed - nice.